### PR TITLE
Fix/htaccess

### PR DESCRIPTION
**Pull request by finnlewis**
@stephen-cox this is the only way I could get the .htaccess file to be in place on a fresh install.

Does this still honour your original motivation of not overwriting .htaccess?

As far as I can see it does. On a composer update I now see this: 

Scaffolding files for localgovdrupal/localgov_microsites_project:
  - Skip [web-root]/.htaccess because it already exists and overwrite is false.

Now then, this would mean that we need to keep the .htaccess file up to date in our assets, so is there another way to get the standard .htaccess to be placed, but not overwritten?

CC @ekes @Adnan-cds @andybroomfield

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -63,8 +63,16 @@
                 "web-root": "web/"
             },
             "file-mapping": {
-                "[web-root]/.htaccess": false,
-                "[web-root]/robots.txt": false,
+                "[web-root]/.htaccess": {
+                    "mode": "replace",
+                    "path": "assets/composer/.htaccess",
```
Comment (stephen-cox) [inline composer.json]: Might be better to have the path as `web/core/assets/scaffold/files/htaccess` and use the core file.

File: composer.json [hunk 2]
```diff
@@ -63,8 +63,16 @@
...
+                    "overwrite": false
+                },
+                "[web-root]/robots.txt": {
+                    "mode": "replace",
+                    "path": "assets/composer/robots.txt",
```
Comment (stephen-cox) [inline composer.json]: Might be better to have the path as `web/core/assets/scaffold/files/robots.txt` and use the core file.

**Review by stephen-cox — CHANGES_REQUESTED** [review]
The basic idea is doing what I wanted (not overwriting changes to .htaccess and robots.txt files).

My suggestion is you use the versions of these files supplied by core rather than supplying our own.

**Comment by finnlewis** [comment]
Done, thanks @stephen-cox  - I'll merge this and tag a release to fix the basic install

**Pull request merged**